dbos/_dbos_config.py

```diff
@@ -141,13 +246,17 @@ def load_config(
 
     """
 
-    init_logger()
-
     with open(config_file_path, "r") as file:
         content = file.read()
         substituted_content = _substitute_env_vars(content)
         data = yaml.safe_load(substituted_content)
 
+    if not isinstance(data, dict):
```

Comment:
Is this check necessary given we're validating the config against the schema?

Reply:
Sorry I didn't see this question: you're right this might be over defensive.